src/Navigators/BottomTab.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { Entypo } from '@expo/vector-icons';
+import { LocalSvg } from 'react-native-svg';
+import Main from '../screens/Main';
+
+const Tab = createBottomTabNavigator();
+
+export default function BottomTabNavigator() {
+    return (
+        <Tab.Navigator
+            screenOptions={({ route }) => ({
+                headerShown: false,
+                tabBarActiveTintColor: '#007DD0',
+                tabBarInactiveTintColor: '#557184',
+                tabBarLabelStyle: {
+                    fontSize: 12,
+                    lineHeight: 14,
+                },
+                tabBarIcon: ({ color, size }) => {
+                    let iconComponent;
+
+                    if (route.name === 'Home') {
+                        iconComponent = <Entypo name="home" size={24} color="#557184" />;
```

Comment:
Is there any reason why we have used Entypo here?

Reply:
I used Entypo to import Expo Vector Icon. However, for the sake of consistency across all the other elements, I have changed it to SVG now.